src/services/tvprogmax/index.jsx

```
@@ -346,92 +348,24 @@ export const TvProgmaxPlayer = ({ video, isOpen, onClose, dispatch }) => {
-            rounded="rounded-full"
-            value={commentText}
-            onChange={(e) => setCommentText(e.target.value)}
-            icon={IoSend}
-            onIconClick={handleComment}
-            onEnterPress={handleComment}
-          />
-        </div>
-      </Modal>
-    </>
+    <VideoPlayer
+      video={video?.video}
```

ESLint

[error] 'video.video' is missing in props validation

(react/prop-types)